### DISCUSSION TITLE
ligatures

### DISCUSSION THREAD

**Discussion by SomeGuyNamedMay** [raphamorim/rio#102] [Ideas]
does this project ever intend to implement them?

**Comment by raphamorim** [comment]
Hey there @SomeGuyNamedMy , by ligatures you mean font ligatures?

If is about font ligatures you should be able to switch fonts in the 0.0.7 version.

**Comment by raphamorim** [comment]
Documentation regarding v0.0.7 release https://raphamorim.io/rio/release-0.0.7/
